db/model/sample.js

```diff
@@ -126,6 +126,8 @@ module.exports = function sample(seq, dataTypes) {
                 'infoRange',
                 'okRange',
                 'valueLabel',
+                "valueType",
```

Comment:
my mistake, I should have noticed this. I'll run the code style check in the future.